<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobotTrajectoryGenerator</class>
 <widget class="qMRMLWidget" name="RobotTrajectoryGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>279</width>
    <height>286</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="outputsCollapsibleButton" native="true">
     <property name="text" stdset="0">
      <string>Outputs</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Lookup:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="qMRMLNodeComboBox" name="lookupSelector" native="true">
        <property name="toolTip">
         <string>Pick the output to the algorithm.</string>
        </property>
        <property name="nodeTypes" stdset="0">
         <stringlist>
          <string>vtkMRMLROS2Tf2LookupNode</string>
          <string>vtkMRMLLinearTransformNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes" stdset="0">
         <bool>false</bool>
        </property>
        <property name="noneEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="addEnabled" stdset="0">
         <bool>true</bool>
        </property>
        <property name="removeEnabled" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="tracePathButton">
     <property name="text">
      <string>Trace path</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="clearPathButton">
     <property name="text">
      <string>Clear path</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="sendPoseArrayButton">
     <property name="text">
      <string>Send trajectory</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>RobotTrajectoryGenerator</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>lookupSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
